examples/pennylane_run_quantum_natural_gradient.py

```diff
@@ -0,0 +1,356 @@
+r"""
+
+.. _quantum_natural_gradient:
+
+Quantum natural gradient
+========================
+
+This example demonstrates the quantum natural gradient optimization technique
+for variational quantum circuits, originally proposed in
+`Stokes et al. (2019) <https://arxiv.org/abs/1909.02108>`__.
+
+Background
+----------
+
+The most successful class of quantum algorithms for use on near-term noisy quantum hardware

```

Comment:
Reading this tutorial in the context of the docs is a bit of an overkill, since the docs explain variational circuits in detail. 

Could we integrate this introduction better? Or shorten it and link statements to the concepts section?

Reply:
> Could we integrate this introduction better?

My current thinking for the QML tutorials section is that they should be a little more self-contained then the remainder of the docs --- they should tell a (high-level) 'story' from start to finish without needing to open additional links, and be relatively independent from the key concepts/API sections. This will also help them work better as a 'point of entry' to a user interested in implementing specific QML algorithms.

As we encourage more submissions from researchers less familiar with ReST roles/the existing documentation structure, this will likely be the best approach to ensure tutorial growth, even if it means there will be some repetition of concepts.

@co9olguy, thoughts?

Reply:
I see, that makes sense. A compromise would be to refer back to the other docs, i.e. start with  

"As laid out in the :ref:` concepts <concepts>` section, the most successful..."

Only if it makes sense to you...